### DISCUSSION TITLE
How can I turn off the sddm for logging.

### DISCUSSION THREAD

**Discussion by FALLfield** [prasanthrangan/hyprdots#999] [Q&A]
I prefer to use the traditional way to enter the hyprland but not sddm. I am not quite sure about the steps to turn off it. Should I make some changes in hyprland too? Or just open the tty when I am in sddm part. It will be helpful!

**Comment by kRHYME7** [comment]
``` systemctl diable sddm ``` or just transfer to other tty session. This repo didn't set anything hyprland related to sddm so it should be fine.
